sst/weatherForecast/Weather_Forecast.ipynb

```diff
@@ -0,0 +1,531 @@
+{
```

Comment:
<p>This seems a cell used for debug. If this is the case, please remove it. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/reports-performance-summary/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Cell removed